elastic/kibana
# [Infra UI] Hosts : Moving to Beta changes

Closes [#159682](https://github.com/elastic/kibana/issues/159682)

## Summary

This PR changes "Technical preview" badges and descriptions to "Beta" and hosts landing page content. By default, the user will see the hosts view (it will be enabled ) and there is still an option to disable it from the advanced settings.


## Testing
- Navigate to hosts
  - The icon in the navigation should be changed to "BETA"
  - Hosts view should be enabled by default 
  - Host view beta badge is updated:
  <img width="1646" alt="image" src="https://github.com/elastic/kibana/assets/14139027/b4609121-ed57-4466-9ed6-258a0eeeb69b">

- Navigate to Inventory
   - The hosts link badge is updated to beta:
   <img width="1654" alt="image" src="https://github.com/elastic/kibana/assets/14139027/70c1a944-9cdd-4650-b6ec-05d2b9629f02">

- As the host view is disabled by default to see the landing page go to [Stack Management] > [Advanced Settings] and disable the host feature (save changes and refresh the page): 
<img width="1649" alt="image" src="https://github.com/elastic/kibana/assets/14139027/c6528496-cde8-42fd-8e82-87e3cc1f7238">

- Go to host view again and check the landing page:
<img width="2493" alt="image" src="https://github.com/elastic/kibana/assets/14139027/19462d15-187b-45c1-8810-e9363ecb98a8">
